pre_commit/error_handler.py

```diff
@@ -26,6 +28,11 @@ def _to_bytes(exc):
 
 def _log_and_exit(msg, exc, formatted):
     error_msg = b''.join((
+        _to_bytes('### version information\n'),
+        _to_bytes('pre-commit.version={}\n'.format(C.VERSION)),
+        _to_bytes('sys.version={}\n'.format(sys.version.replace('\n', ' '))),
```

Comment:
for sys.version, since it can contain newlines, let's instead put the version information on the next line and indent it (instead of removing the newlines)